docs/metrics/object-detection.md

```diff
@@ -0,0 +1,121 @@
+# Object Detection
+
+## Overview of Object Detection
+
+Object detection is a widely applied computer vision task. Object detection aims to locate and identify objects within
+an image or video, typically with bounding boxes.
+
+<figure markdown>
```

Comment:
If we have to pick one image example, I would vote for a multiclass example so it shows both localization and classification aspects of OD.